From f5181d5e01de53f75484c4656c20a0114c19670f Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 9 Dec 2023 16:29:49 -0600
Subject: [PATCH] Disable LTO for libQt5XkbCommonSupport

xdg-desktop-portal-kde builds with this but does not currently use Clang for building. We can remove this once we have toolchain support for recompiling LLVM IR

---
 src/platformsupport/input/xkbcommon/xkbcommon.pro | 1 +
 src/platformsupport/themes/themes.pro             | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/platformsupport/input/xkbcommon/xkbcommon.pro b/src/platformsupport/input/xkbcommon/xkbcommon.pro
index 22b16ae44a..d7dfbdf557 100644
--- a/src/platformsupport/input/xkbcommon/xkbcommon.pro
+++ b/src/platformsupport/input/xkbcommon/xkbcommon.pro
@@ -3,6 +3,7 @@ MODULE = xkbcommon_support
 
 QT = core-private gui-private
 CONFIG += static internal_module
+CONFIG -= ltcg
 
 DEFINES += QT_NO_CAST_FROM_ASCII
 PRECOMPILED_HEADER = ../../../corelib/global/qt_pch.h
diff --git a/src/platformsupport/themes/themes.pro b/src/platformsupport/themes/themes.pro
index 668a843473..6a6179a792 100644
--- a/src/platformsupport/themes/themes.pro
+++ b/src/platformsupport/themes/themes.pro
@@ -3,6 +3,7 @@ MODULE = theme_support
 
 QT = core-private gui-private
 CONFIG += static internal_module
+CONFIG -= ltcg
 
 DEFINES += QT_NO_CAST_FROM_ASCII
 
-- 
2.43.0

